### DISCUSSION TITLE
construct_cis_pairs function

### DISCUSSION THREAD

**Discussion by VeronicaYumiceba** [Katsevich-Lab/sceptre#106] [Q&A] [answered]
Dear Sceptre team, 
I would like to have some clarification regarding "construct_cis_pairs" function 
In the dataset highmoi_example_data, for the candidate_enh_1, the two gRNAs proposed comprise a region in chr1 between 205,720,419-205,720,828. If I would like to get all the genes within 5Mio base pairs I will do the following:

start_overlap <- 205,720,419 - 2,500,000 = 203,220,419
end_overlap <- 205,720,828 + 2,500,000  = 208,220,623

overlap <- gene_position_data_frame_grch38[(gene_position_data_frame_grch38$chr == "chr1" &    gene_position_data_frame_grch38$position > start_overlap  & gene_position_data_frame_grch38$position < end_overlap),]

The overlap variable will have 104 genes, but when I use the command construct_cis_pairs there are only 13 genes present in the discovery pairs associated with candidate_enh_1.  Is this because the pairs are based on the genes from the response matrix? or am I understanding something incorrectly?

**Comment by timothy-barry** [comment, marked as answer]
Hi Veronica,

Yes, the `cis` pairs returned from `construct_cis_pairs` only contain genes that are contained within the `sceptre_object`.

Best,
Tim

**Reply by VeronicaYumiceba** [reply]
Great thanks a lot
